recipes/liblsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Conda build recipe for liblsl
+
+{% set name = "liblsl" %}
+{% set version = "1.14.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sccn/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 196d28608f0c91fb589fd6392e5f4bf2c4d6d5ee4a68ae1df22bf7352a7621b4
+
+{% set common_cmake_args = "-DCMAKE_BUILD_TYPE=Release -DLSL_BUNDLED_PUGIXML=OFF -DLSL_UNIXFOLDERS=ON -S . -B build" %}
+
+build:
+  number: 0
```

Comment:
As a library, this has an ABI that is probably changing from time to time, please add a matching `run_exports` here.

Reply:
liblsl only exports C functions and only imports symbols from the C++ stdlib and pugixml. pugixml is linked in as a static library, so there's only the libgcc / libstdc++ / msvcrt remaining. Would this be sufficient?

```
build:
  run_exports:
    - {{ name }}
```

Reply:
This isn't about what symbols it exports but rather how stable the symbols are. Do you expect people will need to re-compile between `x.x` releases or only between `x` releases?

Reply:
Thank you for the explanation. The next ABI break, whenever that will be, will be in liblsl 2.0, so only between `x` releases. Is this what `{{ pin_subpackage('liblsl', max_pin='x') }}` in `run_exports` does?

Reply:
Yes, exactly!

Reply:
Great, I've added it in the latest commit.